### DISCUSSION TITLE
DÉBARASSER L'ESPACE DE TRAVAIL DE TOUTES SES ICONES

### DISCUSSION THREAD

**Discussion by Tournesol01** [A-S-T-U-C-E/STudio4Education#51] [Ideas]
Autre suggestion

... et finalement, si on débarrassait complétement l'espace de travail de ses icônes, pour que tout l'espace soit réservé... au travail ?
D'autant plus logique qu'une icône dans le bandeau bleu permet de réduire la tool box !

En effet, les icônes «undo», «redo», «ajuster à l'écran», «centrage du sketch», «zoom +» et «zoom -» peuvent très avantageusement figurer dans le bandeau supérieur.
On évite ainsi, comme pour la corbeille, de se trouver dans l'impossibilité d'accéder à l'une ou l'autre de ces icônes parce qu'elle est masquée par des blocs du sketch en cours.

Cela étant dit, on peut même se passer des icônes «undo» et «redo» (si la place manque dans le bandeau).
En effet, «ctrl-Z» et «ctrl-Y» sont des standards de Windows et sont plus rapides à utiliser que les icônes.

En fait, seules les icônes «centrage», «ajuster à l'écran», «zoom -» et «zoom +» sont à conserver et à disposer dans le bandeau supérieur.
Cela ne gênerait pas l'apparition des messages d'identification des fonctionnalités.
(Cela dit, la roulette centrale de la souris est déjà utilisée pour zoomer avant ou arrière)

**Comment by SebCanet** [comment]
Il faut bien avoir en tête que dans la cible d'utilisateur, il y a des enfants de 10 ans, et des élèves avec des difficultés diverses et variées.
Mélanger les informations peut être gênant, là elles sont par lieu : ce qui relève du workspace est dedans, ce qui relève du code est dedans, etc.
L'autre problème est de tout mettre dans la barre du haut et de savoir s'il y a la place...
